nx/lib/nx.ex

```diff
@@ -6302,14 +6302,21 @@ defmodule Nx do
         10
       >
 
-  If a tensor of floats is given, it still returns integers:
+  If a tensor of floats or complex numbers is given, it still returns integers:
 
       iex> Nx.argmax(Nx.tensor([2.0, 4.0]))
       #Nx.Tensor<
         s64
         1
       >
 
+     iex> Nx.argmax(Nx.tensor([Complex.new(1, 2), Complex.new(3, 0)]))
+     #Nx.Tensor<
+       s64
+       0
```

Comment:
This is not exactly what I was expecting, but at least it's consistent with what Nx.greater returns!

If both components are bigger, the comomex is bigger, otherwise it is not